src/main/java/org/sigmah/server/servlet/exporter/template/IndicatorEntryExcelTemplate.java

```diff
@@ -215,6 +215,7 @@ private void createDetailSheet(final IndicatorDTO indicator) {
 		Map<String, Integer> columnIndexMap = new HashMap<String, Integer>();
 		for (PivotTableData.Axis axis : leaves) {
 			utils.putHeader(row, ++cellIndex, axis.getLabel());
+            axis.setLabel(String.valueOf(cellIndex));
```

Comment:
@manishvishnoi2 : This needs an indentation fix before the pull request can be merged.